infra/src/test/kotlin/fi/fta/geoviite/infra/geocoding/GeocodingServiceIT.kt

```diff
@@ -32,38 +32,35 @@ class GeocodingServiceIT @Autowired constructor(
     @Test
```

Comment:
Tästä eteenpäin testikorjauksia. Ei pitäisi olla mitään logiikkamuutoksia, vaan pelkästään poistettu datan alustuksista nuo redundantit viitteet tracknumberiin. Samalla tapahtui vähän formatointia.